common-services-api/hbase/pom.xml

```diff
@@ -67,7 +67,8 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-            <Export-Package>org.pentaho.bigdata.api.hbase,org.pentaho.bigdata.api.hbase.*</Export-Package>
+            <Bundle-Version>${pentaho-osgi-bundles.version}</Bundle-Version>
+            <Export-Package>org.pentaho.bigdata.api.hbase;version=${pentaho-osgi-bundles.version}</Export-Package>
```

Comment:
Missing wildcard

Reply:
Yep. This should be a wildcard. I'll fix it.